common/v2/services/EthService/wallet/non-deterministic/wallets.ts

```diff
@@ -1,8 +1,8 @@
 import { fromPrivateKey, fromEthSale, IFullWallet } from 'ethereumjs-wallet';
 import { fromEtherWallet } from 'ethereumjs-wallet/thirdparty';
 
-import { decryptPrivKey } from 'libs/decrypt';
-import { fromV3 } from 'libs/web-workers';
+import { fromV3 } from 'v2/workers';
+import { decryptPrivKey } from '../../utils';
```

Comment:
Consider writing this as `v2/utils`.

Reply:
changed to  'v2/services/EthService/utils'